scripts/fetch/fetch.precip_shapefiles.R

```diff
@@ -0,0 +1,32 @@
+fetchTimestamp.precip_shapefiles <- vizlab:::fetchTimestamp.file
```

Comment:
for a quick and dirty fix, i recommend
```r
fetchTimestamp.precip_shapefiles <- function(viz) {
  invisible()
}
```
and delete the file at `./vizlab/make/timestamps/precip_shapefiles` if it exists locally

Reply:
i think i'm gonna be looking at `fetchTimestamp.url` a little harder as soon as I get my current PR in, so i can help with the more complete fix (checking remote timestamps, leveraging `vizlab`) then. will loop you in when i do